storage/objects/change_object_csek_to_cmek.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package objects
+
+// [START storage_object_csek_to_cmek]
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+)
+
+// сhangeObjectCSEKtoKMS changes the key used to encrypt an object from
+// a customer-supplied encryption key to a customer-managed encryption key.
+func сhangeObjectCSEKtoKMS(w io.Writer, bucket, object string, encryptionKey []byte, KMSKeyName string) error {
```

Comment:
It's not ideal, but following https://github.com/golang/go/wiki/CodeReviewComments#initialisms and https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps this should have a capital T.
```suggestion
func сhangeObjectCSEKToKMS(w io.Writer, bucket, object string, encryptionKey []byte, KMSKeyName string) error {
```